rust/windows-client/src-tauri/src/client/debug_commands.rs

```diff
@@ -14,13 +15,23 @@ pub enum Cmd {
 
 pub fn run(cmd: Cmd) -> Result<()> {
     match cmd {
+        Cmd::CheckForUpdates => check_for_updates(),
         Cmd::Crash => crash(),
         Cmd::Hostname => hostname(),
         Cmd::NetworkChanges => client::network_changes::run_debug(),
         Cmd::Wintun => wintun(),
     }
 }
 
+fn check_for_updates() -> Result<()> {
+    tracing_subscriber::fmt::init();
+    let rt = tokio::runtime::Runtime::new().unwrap();
+    let release = rt.block_on(client::updates::check());
+    println!("{:?}", release);
```

Comment:
I'll change it, but why not? For debug commands I've been treating them as if they're CLI programs, so checking for an update would print the update info to stdout

Reply:
I could be wrong, but one reason I think it might be because this won't get written to the logger file so that it's auto-uploaded for debugging later?